cpp/CMakeLists.txt

```diff
@@ -658,6 +658,21 @@ set_source_files_properties(
   PROPERTIES COMPILE_DEFINITIONS "_FILE_OFFSET_BITS=64"
 )
 
+set_source_files_properties(
```

Comment:
I don't think you've changed anything in `csv_gpu.cu` that requires CMake alterations. Was there an improperly resolved merge conflict somewhere?